src/sessions.ts

```diff
@@ -172,7 +162,10 @@ export class ClientSession extends TypedEventEmitter<ClientSessionEvents> {
     this.sessionPool = sessionPool;
     this.hasEnded = false;
     this.clientOptions = clientOptions;
-    this[kServerSession] = undefined;
+
+    this.explicit = Boolean(options.explicit);
```

Comment:
I don't really agree on the `!!`, because `!!` is entirely equivalent to `Boolean` and unlike the other examples on the list, there aren't multiple interpretations to what a `!` in front of a variable could mean, it's just shorter. However, if the entire codebase used `Boolean` and you decided to do `!!`, I would give the reverse feedback, because in matters of opinion, the best practice is to opt for *consistency*. I am not opposed to explicit casts in principle, if that's what the entire team prefers, but if we want to make this change in convention, then the change should be applied to the entire codebase and enforced with the lint rule you are referencing.